data/events/2024/campinas/main.yml

```
@@ -0,0 +1,122 @@
+ # - name: program
+ # - name: speakers
+  - name: sponsor
+  - name: organizadores
+  - name: código de conduta
+    url: /events/2024-campinas/conduta 
+  # - name: requisitos para as palestras
+  #   url: /events/2024-campinas/propose 
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Giovana do Nascimento Pena"
+    employer: "IBM"
+    twitter: "GiovanaNp1"
+    bio: "Application Developer Fullstack na IBM, Formada de Analise e Desenvolvimento de Sistemas na Universidade UNASP e Pós-graduanda em DevOps pela PUC Minas, Certificada em Developer Profession - Level 2 Experienced e IBM Cloud. Participante do instituto meninas negras, uma das lideres do TLC a comunidade técnica interna da IBM e Líder da comunidade Elastic São Paulo. Além de organizadora do DevOpsDays São Paulo e Campinas"
+    image: "giovana.png"
+    email: "giovana.np1@gmail.com"
+    pronouns: "ela/dela"
+    role: "Fullstack Developer"
+    github: "https://github.com/GiovanaNp1"
+    linkedin: "https://www.linkedin.com/in/giovana-do-nascimento/"
+    website: "https://giovana-nascimento.netlify.app/"
+
+  - name: "Daniele Pishinin"
+    employer: "Conta Simples"
+    twitter: "danipishinin"
+    bio: "Desenvolvedora mobile, mentora na Reprograma, Embaixadora na Google Women Techmakers, faixa amarela no Krav Maga e iniciante no mundo de trekking e academia."
+    pronouns: "ela/dela"
+    role: "Software Engineer"
+    image: "dani.png"
+    linkedin: "https://www.linkedin.com/in/danipishinin/"
+
+  - name: "Raphael Borges"
+    role: "DevOps"
+    twitter: "RaphaNBorges"
+    bio: "Criador e mantenedor do projeto OKE-FREE: Uma maneira fácil de garantir seu próprio cluster Kubernetes gratuitamente e para sempre na Oracle Cloud Infrastructure (OCI) utilizando OpenTofu. Expert em Container e Kubernetes pelo Programa Intensivo em Containers e Kubernetes (PICK) da Linuxtips. Coautor do livro 'Descomplicando Kubernetes'."
+    image: "raphael.png"
+    github: "https://github.com/Rapha-Borges"
+    linkedin: "https://www.linkedin.com/in/raphael-do-nascimento-borges/"
+    website: "https://r11s.com.br/"
+
+  - name: "Flavio Pimenta"
+    role: "Software Engineer"
+    twitter: "pimentaflavio"
+    bio: "Arquiteto de Soluções focado em traduzir necessidades de negócio em soluções técnicas robustas e com alta qualidade de ponta a ponta.."
+    image: "pimenta.jpeg"
+    github: "https://github.com/flap"
+    linkedin: "https://www.linkedin.com/in/flaap/?originalSubdomain=br"
+    website: "https://campsite.bio/pimenta/"
+    
+organizer_email: "devopsdays.campinas@gmail.com" # Put your organizer email address here
```

This _must_ be your `@devopsdays.org` address.

```suggestion
organizer_email: "campinas@devopsdays.org" # Put your organizer email address here
```